NOpenType/Tables/Glyf.cs

```diff
@@ -95,15 +85,21 @@ private static Glyph ReadSimpleGlyph(BinaryReader input, int count, Bounds bound
             var xs = ReadCoordinates(input, pointCount, flags, Flag.XByte, Flag.XSignOrSame);
             var ys = ReadCoordinates(input, pointCount, flags, Flag.YByte, Flag.YSignOrSame);
 
-            return new Glyph(xs, ys, flags.Select(f => HasFlag(f, Flag.OnCurve)).ToArray(), endPoints, bounds);
+            var onCurves = new bool[flags.Length];
+            for (int i = flags.Length - 1; i >= 0; --i)
+            {
+                onCurves[i] = ((int)flags[i] & 0x1) == 1;
```

Comment:
1. Don't cast to int
2. Bitwise `and` with the flag constant instead like so

`flags[i] & Flags.OnCurve == Flags.OnCurve`